bin/web-push.js

```diff
@@ -0,0 +1,45 @@
+#! /usr/bin/env node
+const webPush = require('web-push');
```

Comment:
Shouldn't this be `require('../index.js')`, so we always import the current version of the lib?